content/spin/v2/language-support-overview.md

```diff
@@ -13,7 +13,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "Rust"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/index.html)
+**[📄 Visit the Rust Spin SDK reference documentation](https://docs.rs/spin-sdk/latest/spin_sdk/) to see specific modules, functions, variables and syntax relating to the following Rust features.**
```

Comment:
For me the document symbol is blurry and hard to parse - it doesn't really draw my attention.  How might it look with something more dramatic like a BIG HONKIN' ARROW or something like that?